### PR TITLE
Apids (bee people ghost role)

### PR DESCRIPTION
**Pull request by kipdotnet**
Apids, aka bee people aka Beeple. they work similarly to the reinforcement items, where cargo can buy an Apid cryopod thatll spawn one of these little guys with a full botanist uniform and PDA and everything. The idea is that you can call one in as a botanist in the event that you have a bunch of plant bounties and theres no botanist on board, instead of just wirehacking the door open and planting them yourself.

https://youtu.be/kFcL4iK0eAU

image

image

**Changelog**

:cl:
- add: Apids, a Beefolk ghostrole that cargo can order in to help botany and medical.

### CODE REVIEW

**Comment by kipdotnet** [comment]
Finished with this! should be good to merge barring any scary things happening

**Comment by kipdotnet** [comment]
image
image

**Comment by kipdotnet** [comment]
doin sprite tweaks please wait warmly

**Comment by kipdotnet** [comment]
done yay

**Comment by kipdotnet** [comment]
image
doctor added on request

**Comment by Darkmajia** [comment]
beeple..
these are adorable, but i wanna be a little nitpicky: i think the apid cryopods could be parented better. its really minor but if they have to be adjusted in any way it would be nicer to just have one prototype to change. it also means you dont have to redefine the sprite for every cryopod
also, small issue i noticed when pressing scream as a bee: names get formatted as "The [name]". i think this can be solved with a grammar component:
```
  - type: Grammar
    attributes:
      proper: true
```

**Comment by kipdotnet** [comment]
oh easy, thank you for the grammar tip bc that was annoying me too. gimme a sec

**Comment by kipdotnet** [comment]
image

should be good, barring the online linter doesnt kill me

File: Resources/Prototypes/_Impstation/Entities/Structures/Specific/apid.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+- type: entity
+  id: ApidCryopodBotanist
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Entities/Structures/Specific/apid.yml:2]: i was thinking of these cryopods when i mentioned the parenting (i did totally gloss over the orders though, good catch)

**Comment by kipdotnet** [comment]
they didnt have the moth accent. this has now been fixed

**Review by Darkmajia — APPROVED** [review]
beeple

**Pull request merged**